packages/sdk/src/react/_internal/transaction-machine/execute-transaction.ts

```diff
@@ -388,19 +389,21 @@ export class TransactionMachine {
 		}
 	}
 
-	async start(props: Input) {
+	async start(props: Input, changeCheckoutIsLoading?: (value: boolean) => void) {
```

Comment:
It makes start methods `correlated` to checkout modal. However it's a part of the `TransactionMachine` and used by listing, offer, selling, cancelling modals.